### ISSUE TITLE
[BUG]: `colorful-winsep` is on top of `lualine`

### ISSUE THREAD

**Issue by ls-devs** [nvim-zh/colorful-winsep.nvim#57]
Hello,
Since latest updates, `colorful-winsep` is on top of `lualine` :
![image](https://github.com/nvim-zh/colorful-winsep.nvim/assets/58638832/4c743516-3ffc-4cee-94f7-6dec7f8526cb)

Config : 
```lua
{
	"nvim-zh/colorful-winsep.nvim",
	event = { "WinNew" },
	opts = {
		hi = {
			fg = "#f5e0dc",
		},
		no_exec_files = {
			"TelescopePrompt",
			"dbui",
			"dbout",
		},
		symbols = { "━", "┃", "┏", "┓", "┗", "┛" },
		smooth = true,
		anchor = {
			left = { height = 1, x = -1, y = -1 },
			right = { height = 1, x = -1, y = 0 },
			up = { width = 0, x = -1, y = 0 },
			bottom = { width = 0, x = 1, y = 0 },
		},
	},
}
```

**Comment by denstiny** [comment]
Sorry, I honestly don't have any more spare time, feel free to pull.

**Comment by nikitarevenco** [comment]
For now if someone wants to fix this you can use an earlier commit, `e1b72c498f25c1fc37a7e9913332c137f753a90a`
I use lazy.nvim

![image](https://github.com/nvim-zh/colorful-winsep.nvim/assets/154856872/da0dd94d-7537-4781-88f4-15621f216f33)

**Comment by denstiny** [comment]
I need vim.opt parameters to reproduce the scene and fix this bug
